browser-config.js

```diff
@@ -34,6 +34,7 @@ module.exports = {
 		"import/extensions": ["error", "ignorePackages"],
 		"import/no-absolute-path": 2,
 		"no-console": ["error", { "allow": ["warn", "error"] }],
+		"no-multi-spaces": "error",
```

Comment:
nit: looks like this is the style mostly used in this config
```suggestion
		"no-multi-spaces": 2,
```